packages/components/src/styled-provider/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import createEmotionCache from '@emotion/cache';
+import { CacheProvider } from '@emotion/core';
+import stylisPluginCssVariables from 'stylis-plugin-css-variables';
```

Comment:
Code originally existed in this repo as part of this PR.
It was abstracted to a separate repo/package:
https://github.com/ItsJonQ/stylis-plugin-css-variables